### PR TITLE
release 1.10 updates

### PR DESCRIPTION
**Pull request by jahrlin**
Signed-off-by: Joakim Ahrlin <joakim.ahrlin@gmail.com>

### CODE REVIEW

**Review by SgtCoDFish — CHANGES_REQUESTED** [review]
One comment which I think might need addressing before we can merge! Looks great overall though!

File: content/v1.10-docs/manifest.json [hunk 1]
```diff
@@ -0,0 +1,709 @@
...
+        },
+        {
+            "title": "Getting Started",
+            "path": "/docs/getting-started/README.md"
+        },
+        {
+          "title": "Installation",
+          "routes": [
+            {
+              "title": "Introduction",
```
[the remark this answers is not in the document]
Comment (jahrlin) [reply content/v1.10-docs/manifest.json]: Good spot! Fixed 👍

File: content/v1.10-docs/manifest.json [hunk 2]
```diff
@@ -5,179 +5,179 @@
...
             {
               "title": "Upgrading",
               "routes": [
                 {
                   "title": "Introduction",
-                  "path": "/docs/installation/upgrading/README.md"
+                  "path": "/v1.10-docs/installation/upgrading/README.md"
                 },
                 {
                   "title": "Notes on Ingress Class Compatibility",
```
Comment (SgtCoDFish) [inline content/v1.10-docs/manifest.json:65]: suggestion: sorry, I missed this first time through. There are actually a bunch of docs which we don't include for versions, including "upgrading", "contributing" and release notes.

E.g. in https://cert-manager.io/v1.8-docs and https://cert-manager.io/v1.9-docs

Removing those would also reduce the size of this PR by a decent amount too!

This is another of those things which should be automated but isn't :sob:

File: content/v1.10-docs/installation/helm.md [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+
+To install the cert-manager Helm chart, use the [Helm install command](https://helm.sh/docs/helm/helm_install/) as described below.
+
+```bash
+helm install \
+  cert-manager jetstack/cert-manager \
+  --namespace cert-manager \
+  --create-namespace \
+  --version v1.9.1 \
+  # --set installCRDs=true
```
Comment (SgtCoDFish) [inline content/v1.10-docs/installation/helm.md:69]: suggestion: (another thing which should be automated :sob: ) there are some hardcoded versions in installation docs which might need bumping to reference the latest version, in `/v1.10-docs` and also in `/docs` and `/next-docs`.

File: content/v1.10-docs/installation/helm.md [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+A full list of available Helm values is on [cert-manager's ArtifactHub page](https://artifacthub.io/packages/helm/cert-manager/cert-manager).
+
+The example below shows how to tune the cert-manager installation by overwriting the default Helm values:
+
+```bash
+helm install \
+  cert-manager jetstack/cert-manager \
+  --namespace cert-manager \
+  --create-namespace \
+  --version v1.9.1 \
```
Comment (SgtCoDFish) [inline content/v1.10-docs/installation/helm.md:81]: suggestion: another hardcoded version!

File: content/v1.10-docs/installation/helm.md [hunk 5]
```diff
@@ -0,0 +1,206 @@
...
+```yaml
+apiVersion: v2
+name: example_chart
+description: A Helm chart with cert-manager as subchart
+type: application
+version: 0.1.0
+appVersion: "0.1.0"
+dependencies:
+  - name: cert-manager
+    version: v1.9.1
```
Comment (SgtCoDFish) [inline content/v1.10-docs/installation/helm.md:111]: suggestion: another hardcoded version!

File: content/v1.10-docs/installation/helm.md [hunk 6]
```diff
@@ -0,0 +1,206 @@
...
+
+Instead of directly installing cert-manager using Helm, a static YAML manifest can be created using the [Helm template command](https://helm.sh/docs/helm/helm_template/).
+This static manifest can be tuned by providing the flags to overwrite the default Helm values:
+
+```bash
+helm template \
+  cert-manager jetstack/cert-manager \
+  --namespace cert-manager \
+  --create-namespace \
+  --version v1.9.1 \
```
Comment (SgtCoDFish) [inline content/v1.10-docs/installation/helm.md:142]: suggestion: another hardcoded version!

File: content/v1.10-docs/installation/kubectl.md [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+
+### Steps
+
+All resources (the [`CustomResourceDefinitions`](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions) and the cert-manager, cainjector and webhook components)
+are included in a single YAML manifest file:
+
+Install all cert-manager components:
+
+```bash
+kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.9.1/cert-manager.yaml
```
Comment (SgtCoDFish) [inline content/v1.10-docs/installation/kubectl.md:22]: suggestion: another hardcoded version!

File: content/v1.10-docs/installation/README.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+install methods are listed below for each of the situations.
+
+## Default static install
+
+> You don't require any tweaking of the cert-manager install parameters.
+
+The default static configuration can be installed as follows:
+
+```bash
+kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.9.1/cert-manager.yaml
```
Comment (SgtCoDFish) [inline content/v1.10-docs/installation/README.md:17]: suggestion: another hardcoded version!

**Review by SgtCoDFish — CHANGES_REQUESTED** [review]
I remembered a few other bits which need changing. Most of this should be automated so I feel annoying for pointing it out, but I think it might need updating - sorry!

**Comment by jahrlin** [comment]
No worries, thanks for pointing it out!

File: content/v1.10-docs/installation/upgrading/README.md [hunk 9]
```diff
@@ -0,0 +1,121 @@
+---
+title: Upgrading
+description: 'cert-manager installation: Upgrading cert-manager overview'
+---
```
Comment (SgtCoDFish) [inline content/v1.10-docs/installation/upgrading/README.md:4]: suggestion: I think this whole upgrading folder can be removed!

File: content/v1.10-docs/installation/supported-releases.md [hunk 10]
```diff
@@ -0,0 +1,270 @@
+---
+title: Supported Releases
+description: Supported releases, Kubernetes versions, OpenShift versions and upcoming release timeline
+---
+
+{/*
+Inspired by https://istio.io/latest/about/supported-releases/
```
Comment (SgtCoDFish) [inline content/v1.10-docs/installation/supported-releases.md:7]: suggestion: supported releases is another thing we don't copy to versioned docs (it probably shouldn't be under "installation" in the first place), so this file can be deleted and removed from the manifest!

File: content/v1.10-docs/manifest.json [hunk 11]
```diff
@@ -0,0 +1,363 @@
...
+          "title": "Installation",
+          "routes": [
+            {
+              "title": "Introduction",
+              "path": "/v1.10-docs/installation/README.md"
+            },
+            {
+              "title": "Supported Releases",
+              "path": "/v1.10-docs/installation/supported-releases.md"
+            },
```
Comment (SgtCoDFish) [inline content/v1.10-docs/manifest.json:24]: suggestion: as mentioned before, this part can be removed

**Review by SgtCoDFish — CHANGES_REQUESTED** [review]
A few more little bits I spotted; nearly there :grin:

**Review by SgtCoDFish — APPROVED** [review]
/lgtm
/approve

Thanks very much! So much potential for automation here but as with so much that'll have to wait for another time :grin:

**Pull request merged**